### PR TITLE
[ansible/fanout] Update ansible files of deploy fanout

### PR DESCRIPTION
**Pull request by lizhijianrd**
### Description of PR

Summary:
1. Fixed the management IP in `ansible/roles/fanout/templates/arista_7060_deploy.j2`.
2. Add an ansible task to deploy fanout which is using SONiC 202205 image. 

### Type of change

- [x] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?

1. Fixed the management IP in `ansible/roles/fanout/templates/arista_7060_deploy.j2`.
2. Add an ansible task to deploy fanout which is using SONiC 202205 image. 

#### How did you do it?

Updated ansible playbooks and template files.

#### How did you verify/test it?

Verified on the following physcal testbeds:
- Leaf fanout `Arista-720DT-48S` with SONiC master image.
- Leaf fanout `Arista-7060CX-32S` with Arista EOS.

### CODE REVIEW

File: ansible/roles/fanout/tasks/fanout_sonic.yml [hunk 1]
```diff
@@ -21,3 +21,8 @@
   include_tasks:
     sonic/fanout_sonic_202012.yml
   when: "'20201231' in fanout_sonic_version['build_version']"
+
+- name: deploy SONiC fanout with image version master
+  include_tasks:
+    sonic/fanout_sonic_202012.yml
```
Comment (lizhijianrd) [inline ansible/roles/fanout/tasks/fanout_sonic.yml:27]: Do we need to create a new file named `fanout_sonic_master.yml`?

Comment (Blueve) [reply ansible/roles/fanout/tasks/fanout_sonic.yml:27]: master is not stable enough, let's use `202205` instead

File: ansible/roles/fanout/templates/arista_7060_deploy.j2 [hunk 2]
```diff
@@ -121,7 +121,7 @@ interface {{ intf }}
 !
 interface Management 1
  description TO LAB MGMT SWITCH
- ip address {{ device_info[inventory_hostname]["ManagementIp"] }}/{{ hostvars[inventory_hostname].mgmt_subnet_mask_length }}
+ ip address {{ device_info[inventory_hostname]["ManagementIp"] }}
```
Comment (Blueve) [inline ansible/roles/fanout/templates/arista_7060_deploy.j2:124]: Need to figure out why this is a prefix in eos template. The value in sonic template is not a prefix.

File: ansible/roles/fanout/tasks/fanout_sonic.yml (hunk 1 above)
Comment (lizhijianrd) [reply ansible/roles/fanout/tasks/fanout_sonic.yml]: Updated to `202205`

**Review by Blueve — APPROVED** [review]

**Pull request merged**